pysqldb3/geopackage.py

```diff
@@ -63,28 +67,27 @@ def geopackage_exists(self, gpkg_name = None):
                                   
         return gpkg_exists
     
-    def geopackage_tbl_exists(self, gpkg_tbl):
+    def gpkg_tbl_exists(self, gpkg_tbl):
                
         """
         Checks if a geopackage table already exists, in case it isn't meant to be overwritten.
         """
-
-        try:
-            exists_cmd = f'ogrinfo {os.path.join(self.path, self.gpkg_name)}'
-            ogr_response = subprocess.check_output(exists_cmd, stderr=subprocess.STDOUT)
-            table_exists = re.findall(f"{gpkg_tbl}", str(ogr_response)) # only allows tables names with underscores, numbers, and letters
         
-        except:
-            pass 
-        
-        if len(table_exists) == 0:
-            gpkg_tbl_exists = False
+        if self.gpkg_exists():
```

Comment:
I only ever run this function when I've already confirmed that the geopackage exists, but I wanted to build in an extra check just in case. I will just set it to False if no gpkg_exists